lib/spack/spack/cmd/ci.py

```diff
@@ -562,7 +562,7 @@ def ci_rebuild(args):
             "SPACK={}".format(args_to_string(spack_cmd)),
             "SPACK_COLOR=always",
             "SPACK_INSTALL_FLAGS={}".format(args_to_string(deps_install_args)),
-            "-j$(nproc)",
```

Comment:
can this be an environment var and change it for power only? I would hate to loose the parallelism in all of the other CI runners just because the power runners are having issues.

```bash
SPACK_CI_PARALLELISM=auto #nproc
SPACK_CI_PARALLELISM=2
```

Reply:
Hm, `nproc` isn't great, a new variable isn't great either :p and the existing build jobs variable is bad too cause it would be sequential downloading on `small` jobs?

I *think* it's not going to be a huge difference between say -j4 and -j2. But there's likely a larger difference between -j2 and -j1.